src/models/components.jl

```diff
@@ -36,3 +36,7 @@ function get_value(c::T, value::V) where {T <: Component, V}
     @warn("conversion not implemented for $(V) in component $(T)")
     return value::V
 end
+
+function has_forecasts(d::Component)
```

Comment:
This already exists in src/models/devices.jl

Reply:
Is the same function I just moved to the file for components.